applications/IgaApplication/tests/cpp_tests/test_support_laplacian_condition.cpp

```diff
@@ -0,0 +1,99 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License: BSD License
+//           Kratos default license: kratos/license.txt
+//
```

Comment:
Copy and paste the header from any of the core files to keep the same formatting.